release/edgex-taf.yaml

```diff
@@ -1,8 +1,9 @@
 ---
 name: 'edgex-taf'
-version: '1.3.0'
+version: '1.3.1'
 releaseName: 'hanoi'
 releaseStream: 'master'
```

Comment:
Not sure we want/need to tag TAF since `master` has moved way beyond Hanoi and will fail if range against 1.3.1. I assume @cloudxxx8 and team manually ran TAF from the 1.3.0 tag.